### PR TITLE
Change remove page by number to page by title

### PR DESCRIPTION
**Pull request by Lusi711**

### CODE REVIEW

File: src/main/java/seedu/duke/userinterface/command/notebook/AddCommandNotebookMode.java [hunk 1]
```diff
@@ -50,6 +51,9 @@ public void execute() {
                 System.out.println("Added section with title : " + title);
                 break;
             case NOTEBOOK_SECTION:
+                if (content == "") {
+                    throw new EmptyPageException();
+                }
```
Comment (franceneee) [inline src/main/java/seedu/duke/userinterface/command/notebook/AddCommandNotebookMode.java:56]: wait i catch it somewhere else actually o.o but okay

File: src/main/java/seedu/duke/notebooks/Section.java [hunk 2]
```diff
@@ -43,6 +43,15 @@ public void addPage(Page p) {
...
 
+    public void getPage(int pageIndex) {
+        try {
+            Page page = pageArrayList.get(pageIndex);
+            page.printPage();
+        } catch (IndexOutOfBoundsException | NullPointerException e) {
+            System.out.println("page <" + (pageIndex + 1) + "> doesn't exist");
+        }
+    }
+
```
Comment (franceneee) [inline src/main/java/seedu/duke/notebooks/Section.java:54]: do we still need this, now that we're using page title?

Comment (yAOwzers) [reply src/main/java/seedu/duke/notebooks/Section.java:54]: True

Comment (NeilBaner) [reply src/main/java/seedu/duke/notebooks/Section.java:54]: We can just mark as `@Deprecated`

File: src/main/java/seedu/duke/notebooks/Section.java [hunk 3]
```diff
@@ -43,6 +43,15 @@ public void addPage(Page p) {
         pageArrayList.add(p);
     }
 
+    public void getPage(int pageIndex) {
```
Comment (NeilBaner) [inline src/main/java/seedu/duke/notebooks/Section.java:46]: To mark as `@Deprecated`.

**Review by NeilBaner — APPROVED** [review]
I can add the tag later.

**Pull request merged**